src/components/views/rooms/RoomHeader.tsx

```diff
@@ -733,6 +734,7 @@ export default class RoomHeader extends React.Component<IProps, IState> {
                     { betaPill }
                     { buttons }
                 </div>
+                <RoomCallBanner roomId={this.props.room.roomId} />
```

Comment:
Could avoid having RoomCallBanner depend on beacon store by doing something like:

```
// banners in order of priority, only one banner rendered at a time
const prioritisedBanners = [
    <RoomLiveShareWarning roomId={this.props.room.roomId} />,
    <RoomCallBanner roomId={this.props.room.roomId} />
].filter(Boolean);
const banner = prioritisedBanners[0] || null;
```

Would keep things simpler in RoomCallBanner, and make this more future proof

Reply:
That makes sense. It would make the tests involve the whole Room Header though.

Reply:
Do you think it is fine leaving it like this in favor of not changing the tests?